hz: 10                   # ループ周期 [Hz]
laser_step: 3            # 何本ずつレーザを見るか(全1080本)

# --- 柱に関する角度範囲の配列 [rad] ---
# 以下が無視される
# [0] < angle < [1], [2] < angle
# [num] -> num:index
ignore_angle_range_list: [rad(1.5*pi/16.0), rad(5.0*pi/16.0), rad(10.0*pi/16.0)]
# --- はずれ値対策 ---
ignore_dist: 0.01
